PowerShell_Scripts/NameSurname_1more.ps1

```diff
@@ -0,0 +1,8 @@
+﻿Clear-Host
+$name = Read-Host 'What is your name?'
+$surName = Read-Host 'What is your surname?'
+Write-Host 'Indicate the file, where we need to save it'
```

Comment:
Pro-hint: do you think you can re-use the `Path` function from `FolderSynchronisation_edited+func_2ed.ps1` here? 😉

Reply:
For now I may assume something like this:
```suggestion
function GetValue {

    $value = Read-Host

    return $value
}

Write-Host 'What is your name?'
$name = GetValue  
Write-Host 'What is your surname?'
$surName = GetValue 
Write-Host 'Indicate the file, where we need to save it'
$path = GetValue
```
Could you please give me one more hint: is there a possibility to include Write-Host part into the function?

Reply:
Sure. For example, you can add, say, `Prompt` parameter to the function like this:
```
function Get-String()
{
    param(
        [Parameter(Mandatory=$true)]
        [string] $Prompt
    )

    Write-Host $Prompt -NoNewLine
    $value = Read-Host

    return $value
}
```

and update your calls like this:
`$surname = Get-String -Prompt "Enter your surname: "`